CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.3.1] - 2022-03-15
+
+### Added
+
+- Add NodeAffinity to run the exporter only on Linux Nodes with AMD64.
+
```

Comment:
The release heading will be added automatically
```suggestion
### Added

- Add NodeAffinity to run the exporter only on Linux Nodes with AMD64.

```

Reply:
I tested it on EKS today. Thanks for the fast feedback